tests/storage/outofprocess_test.py

```diff
@@ -49,6 +51,23 @@ def oop_cleanup():
     oop.stop()
 
 
+@pytest.fixture(
+    scope="function",
+    params=[
+        userstorage.PATHS["mount-512"],
+        userstorage.PATHS["mount-4k"],
+    ],
+    ids=str,
+)
+def user_mount(request):
+    storage = request.param
+    if not storage.exists():
+        pytest.xfail("{} storage not available".format(storage.name))
+    file = Path(storage.path) / "file"
+    yield file
+    file.unlink()
+
```

Comment:
I think what we should have here is:

    tmpdir = os.mkdir(storage.path, "tmp")
    yield tmpdir
    shutil.rmtree(tmpdir)

The test will use the directory in the same way they use it today.